ain-blockchain/developer-guide/getting-started.md

```diff
@@ -67,340 +73,240 @@ console.log(ain.wallet.defaultAccount);
 
-// Print defaultAccount (Need to backup your private key)
-console.log(ain.wallet.defaultAccount);
-// Example:
-// {
-//   address: '0x09A0d53FDf1c36A131938eb379b98910e55EEfe1',
-//   private_key: '...',
-//   public_key: '...'
-// }
+#### Step 1: Import $AIN Tokens into MetaMask
 
-// Transfer 1 AIN to ADDRESS_TO_SEND_AIN_TO
-ain.wallet
-  .transfer({
-    from: myAddress, // Optional (defaultAccount will be used if omitted)
-    to: '0x99bBa0051DDdf7b69972602512661915fdD8eE89', // or your address
-    value: 1,
-  })
-  .then((res) => {
-    // The returned value will be either
-    // { result, tx_hash } or { code, message, tx_hash },
-    // depending on the result of the transaction.
-    console.log(JSON.stringify(res));
-  });
-```
+1. Open MetaMask and go to “Assets.”
+2. Click “Import Token” and select “Custom Token.”
+3. Use the contract address: `0x3a810ff7211b40c4fa76205a14efe161615d0385`
+4. Click “Add Custom Token” to see your $AIN balance.
 
-{% endcode %}
+#### Step 2: Deposit $AIN to AIN Credits
 
-## Step 6. (Optional) Setting the nonce of your transaction
+1. Type `/ain deposit` in the AIN DAO Discord to receive a unique deposit address (e.g. start with `0x...`).
+2. Copy the deposit address.
+3. Open your ETH wallet and locate the $AIN token.
+4. Click “Send,” paste the copied address, and enter the amount of $AIN to deposit.
+5. Confirm the transaction in your wallet.
+6. In Discord, type `/ain balance` to confirm your deposit.  
+   **Note**: Deposits under 500 $AIN will not be paid out until the total deposited amount exceeds 500 $AIN.
 
-Optionally, you can set the nonce of your transaction. In Step 4, we didn't set the nonce value, which means that ain-js automatically set the transaction's nonce as -1. A transaction with a nonce of -1 is regarded to be "unordered".
+#### Step 3: Withdraw AIN Credits to $AIN
 
-Call `getNonce()` method to get the current nonce of your address and make the same transfer call as in Step 4, but with a nonce that's incremented by 1 from the value you just got from `getNonce()`.
+1. Use the `/ain withdraw` command in Discord.
+2. Select the chain for withdrawal (currently, only Ethereum is supported).
+3. Enter your Ethereum wallet address.
+4. Specify the amount of AIN credits to withdraw (minimum: 10 AIN).
+5. Confirm the transaction.
 
-{% code title="transfer.js" %}
+#### Summary
 
-```javascript
-const Ain = require('@ainblockchain/ain-js').default;
-const ain = new Ain('https://testnet-api.ainetwork.ai', 'wss://testnet-event.ainetwork.ai', 0);
+- **Deposit**: Move your ERC-20 AIN from Ethereum into Discord to convert them into Discord Credits.
+- **Withdraw**: Convert Discord Credits into Native AIN and transfer them to your AI Network wallet.
 
-// Create 1 new account
-const accounts = ain.wallet.create(1);
-const myAddress = accounts[0];
+👉 [Join the AIN DAO Discord](https://discord.com/invite/aindao) to start the process!
```

Comment:
I think it would be better to place the invitation link at the beginning of the section rather than here.
https://github.com/ainblockchain/docs/pull/4/files#diff-99e84eb9e9f6e3696ac8ac30b04c4ffcd4c9c7f62790e08ef12ebbef7144bebbR90

Reply:
thanks~